pghoard/pghoard.py

```diff
@@ -98,6 +98,9 @@ def __init__(self, config_path):
         self.log.info("pghoard initialized, own_hostname: %r, cwd: %r", socket.gethostname(), os.getcwd())
 
     def check_pg_versions_ok(self, pg_version_server, command):
+        if pg_version_server is None:
```

Comment:
should we log something here to say fetching server version number failed?  or do we log that elsewhere?